tools/accuracy_checker/accuracy_checker/evaluators/quantization_model_evaluator.py

```diff
@@ -105,7 +105,11 @@ def _set_number_infer_requests(nreq):
         _set_number_infer_requests(nreq)
 
         if check_progress:
-            progress_reporter = ProgressReporter.provide('print', self.dataset.size)
+            pr_kwargs = {}
```

Comment:
please move progress reporter generation in the function
Possibly it also make sense to rename this parameter to `progress_interval` and made default value is 0. it helps to avoid complicated logic and multiple meaning for argument.

Reply:
But possibly no, it can requires some changes on your side...
or you can stay this bool flag as is and set additional parameter in this method kwargs for progress interval if it is necessary